src/js/modules/core/services/sweeper/sweeper.service.js

```diff
@@ -102,30 +97,58 @@
             }
 
             return bitcoinDataClient.estimateFee().then(function (feePerKB) {
-                if (totalValue <= blocktrailSDK.DUST) {
-                    return false;
-                }
 
-                rawTransaction.addOutput(options.recipient, totalValue);
+                var txAfter = 600;
```

Comment:
Maybe add a comment describing where this number came from (n148+56 I think), it depends on the utxo type never changing, etc